devicetypes/smartthings/zwave-siren.src/zwave-siren.groovy

```diff
@@ -464,6 +482,12 @@ private Boolean secondsPast(timestamp, seconds) {
 	return (new Date().time - timestamp) > (seconds * 1000)
 }
 
+def calculateLength(int alarmLength){
+	//If the siren is Everspring then the alarm length can be set to 1, 2 or max 3 minutes
+	def map = [1:60, 2:120, 3:180]
+	if (alarmLength > 3) return everspringDefaultAlarmLength else return map[alarmLength].value
```

Comment:
Please use either a ternary operator here, or break this if/else statement into multiple lines.